cli/Cargo.toml

```diff
@@ -6,26 +6,26 @@ edition.workspace = true
 license.workspace = true
 homepage.workspace = true
 repository.workspace = true
-description = "CLI tool for runtime testing and dry-running"
+description = "Substrate's programmatic testing framework for power users."
 
 [[bin]]
-name = "try-runtime-cli"
+name = "try-runtime"
 path = "main.rs"
 
 [package.metadata.docs.rs]
 targets = ["x86_64-unknown-linux-gnu"]
 
 [dependencies]
```

Comment:
it'd be nice to have workspace dependencies and properties (like https://github.com/paritytech/ink/pull/1835); if you agree on using GH issues, I can create one for that

Reply:
Sounds great, and yes let's use GH issues https://github.com/paritytech/try-runtime-cli/issues/20